File: ljh/java/src/level1/test/CalculateAverageTest.java
package level1.test;

import org.junit.Before;
import org.junit.Test;

import level1.CalculateAverage;

public class CalculateAverageTest {
	private int[][] arr;
	private double[] answer;
	private CalculateAverage average;
	
	@Before
	public void setUp() throws Exception {
		arr = new int[][] {{1,2,3,4}, {5,5}};
		answer = new double[] {2.5, 5};
		average = new CalculateAverage();
	}
	
	@Test
	public void test() {
		for(int i=0; i<arr.length; i++) {
			assert(average.solution(arr[i]) == answer[i]);
		}
	}
	
	@Test
	public void test2() {
		for(int i=0; i<arr.length; i++) {
			assert(average.solution2(arr[i]) == answer[i]);
		}
	}
}

File: ljh/javascript/level1/test/CalculateAverage.test.js
import CalculateAverage from '../CalculateAverage';
test(`평균 구하기`, ()=>{
    expect(new CalculateAverage().solution([1,2,3,4])).toEqual(2.5);
})

test(`평균 구하기`, ()=>{
    expect(new CalculateAverage().solution([5,5])).toEqual(5);
})